add include and lib directories to find system SDL2
try to avoid get_target_property error

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -2,6 +2,9 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.4)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+include_directories( "${LOCALBASE}/include" "${LOCALBASE}/include/SDL2" )
+link_directories( "${LOCALBASE}/lib" )
+
 # Make sure the source and binary directory are not the sample
 if ("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
 	message(FATAL_ERROR "Binary and source directory can not be the same! Choose another build directory.")
@@ -14,6 +17,7 @@ include(cmake/embed_file.cmake)
 set_policy(CMP0051 NEW)
 set_policy(CMP0053 NEW)
 set_policy(CMP0054 NEW)
+set_policy(CMP0026 OLD)	# to avoid get_target_property error
 
 SET(FSO_PROJECT_DESCRIPTION "Open source project based on the original FreeSpace 2 source code.")
 
